server/src/main/scala/org/scalaexercises/evaluator/services.scala

```diff
@@ -116,12 +113,14 @@ object EvaluatorServer extends IOApp {
   lazy val port = (Option(System.getenv("PORT")) orElse
     Option(System.getProperty("http.port"))).map(_.toInt).getOrElse(8080)
 
+  lazy val evaluator = new Evaluator[IO](15 seconds)
```

Comment:
Out of curiosity, why did we change from 20 seconds to 15 seconds?

Reply:
Oops, should be 20? Or 15 secs is fine?

Reply:
15 should be fine, but ideally, this should be on the application.conf file